recipes/libbigwig/all/conanfile.py

```diff
@@ -91,12 +83,20 @@ def package(self):
         cmake.install()
 
     def package_info(self):
-        self.cpp_info.libs = ["BigWig"]
-        self.cpp_info.system_libs = ["m"]
         self.cpp_info.set_property("cmake_file_name", "BigWig")
         self.cpp_info.set_property("cmake_target_name", "BigWig::BigWig")
+        self.cpp_info.libs = ["BigWig"]
+        self.cpp_info.system_libs = ["m"]
+
+        self.cpp_info.requires = []
+        if self.options.get_safe("with_curl"):
+            self.cpp_info.requires.append("libcurl::libcurl")
+        if self.options.get_safe("with_zlibng"):
+            self.cpp_info.requires.append("zlib-ng::zlib-ng")
+        else:
+            self.cpp_info.requires.append("zlib::zlib")
```

Comment:
The ``requires`` information is only necessary when dealing with components. This package has no components, and is not requiring specific components either, which should be the default behavior already.
```suggestion
```